### PR TITLE
php55, psr4, travis

### PR DESCRIPTION
**Pull request by basz**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3",
+        "php": ">=5.5",
```
Comment (bakura10) [inline composer.json]: We will likely need to upgrade dependency of SlmQueue too. That's why I'd love if we could start our work on SlmQueue 0.5 so we can tag sometimes and upgrade all the other modules.

Comment (bakura10) [inline composer.json]: For more info: https://github.com/juriansluiman/SlmQueue/issues/166

Comment (basz) [inline composer.json]: Are u saying I should not bumping requirement in this PR until 0.5 is done? Or just pointing out 0.5 work is still ahead?

Comment (bakura10) [inline composer.json]: I mean, I don't think it makes sense to bump all adapters to 0.5 but not the main lib (as if we tag this as 0.5, then tag slmqueue as 0.5, the adapters would need to be tagged 0.6).

Envoyé de mon iPhone

**Comment by basz** [comment]
@bakura10 review, merge & tag?

**Comment by bakura10** [comment]
Didn't spot anything wrong :). I'm tagging as 0.5 in a few minutes.

**Pull request merged**